app/src/main/java/com/bignerdranch/android/geoquiz/QuizViewModel.kt

```diff
@@ -21,6 +22,15 @@ class QuizViewModel : ViewModel() {
     val currentQuestionText: Int
         get() = questionBank[currentIndex].textResId
 
+    val isCurrentQuestionAnswered: Boolean
+        get() = questionBank[currentIndex].isAnswered
+
+    val questionBankSize: Int
+        get() = questionBank.size
+
+    val questionBankAreAnswered: Boolean
+        get() = questionBank.all { it.isAnswered }
```

Comment:
organize your `val`s and `private val`s here. You want to keep them grouped, i.e all `private`s together, and all publics together. It's easier to then keep track of what is what

Reply:
Organized them